dev/devicelab/lib/framework/apk_utils.dart

```diff
@@ -29,6 +30,50 @@ final List<String> baseApkFiles = <String> [
   'AndroidManifest.xml',
 ];
 
+
+const String gradleSettingsFileContent = r'''
```

Comment:
Please add dart doc.

Reply:
Added

Reply:
Also added dart docs to the main test methods